README.md

```diff
@@ -1,6 +1,30 @@
 # Funnel
 
 A little CI server for git hygiene. Written in Elixir.
+
+Rationale: [Always Squash and Rebase your Git Commits](https://blog.carbonfive.com/2017/08/28/always-squash-and-rebase-your-git-commits/)
+
+## Using funnel on a project
+
+1. Visit the Funnel homepage: [https://github.com/apps/funnel](https://github.com/apps/funnel)
+
+2. Click `Configure`
+
+3. Select the organization and project(s) you want to use Funnel on
+
+4. Go to the `Settings` for each project you want to use Funnel on and click onto the `Webhooks` section
+
+5. Click `Add webhook`, then enter the following values:
```

Comment:
is this still necessary? it shouldn't be. @hannahhoward have you tried using funnel without these manually setup web hooks?

Reply:
i haven't. oh let me try.